addons/website/models/ir_module_module.py

```diff
@@ -515,7 +515,7 @@ def _load_module_terms(self, modules, langs, overwrite=False):
                 for lang, generic_term_lang in generic_translation_dictionary[term_en].items():
                     if overwrite or term_en == specific_term_langs[lang]:
                         specific_term_langs[lang] = generic_term_lang
-            for lang in langs_update:
+            for lang in specific_langs:
```

Comment:
Is it necessary? I think you only need to update values for `langs_update` and leave other translations as they were